### PR TITLE
Additional fields

### PR DESCRIPTION
**Pull request by feed3r**
Extract two of Elastic Fields (ID and Index)

### CODE REVIEW

**Comment by feed3r** [comment]
Feature has been implemented along with tests. 

Since the new object now contains two Elastic proprietary fields (index and ID) all the tests has been refactored since we could not use the simple toString() comparison and we compared the precise data fields value.

In addition, another thing has been changed in test: the check for list length correctness (I insert two elements, I check that the length is exactly two) has been executed first. I think this is more correct because if (for any reason) this test fails (let's assume that the list contains one only element) the access to the second element would fail (maybe with an exception?) and the test with assert would not be executed. 

Hope this can be a contribution.

File: src/test/java/com/github/dariobalinzo/task/ElasticSourceTaskTest.java [hunk 1]
```diff
@@ -339,8 +392,7 @@ public void shouldRunSourceTaskWhitelist() throws IOException, InterruptedExcept
         //when (fetching first page)
         task.start(conf);
         List<SourceRecord> poll1 = task.poll();
-        assertEquals("Struct{fullName=Test}", poll1.get(0).value().toString());
-
+        assertEquals(((Struct) poll1.get(0).value()).get("fullName"), "Test");
         task.stop();
```
Comment (DarioBalinzo) [inline src/test/java/com/github/dariobalinzo/task/ElasticSourceTaskTest.java:396]: may you please assert here that poll1.get(0).value() size is 1 (to be sure that the whitelist filter is working and emitting only the fullname field, with no additional fields).

File: src/test/java/com/github/dariobalinzo/task/ElasticSourceTaskTest.java [hunk 2]
```diff
@@ -364,8 +416,9 @@ public void shouldRunSourceTaskWithJsonCastFilter() throws IOException, Interrup
         //when (fetching first page)
         task.start(conf);
         List<SourceRecord> poll1 = task.poll();
-        assertEquals("Struct{fullName=\"Test\",nonavrofield=non-avro-field,avroField=avro-field,age=10,ts=111}", poll1.get(0).value().toString());
-
+        Struct structValue = (Struct) poll1.get(0).value();
+        assertEquals(structValue.get("avroField"), "avro-field");
+        assertEquals(structValue.get("nonavrofield"), "non-avro-field");
         task.stop();
```
Comment (DarioBalinzo) [inline src/test/java/com/github/dariobalinzo/task/ElasticSourceTaskTest.java:422]: Similar here, the string fullname is being cast to json: so we need to check that is actual a string literal beginning and ending by `"` (`fullName=\"Test\"`)

**Comment by DarioBalinzo** [comment]
Thanks @feed3r, LGTM!
only two assert are missing in a couple of tests (see comment above).

**Comment by feed3r** [comment]
I fixed both the requested, please check it out and let me know for any other comments

**Comment by DarioBalinzo** [comment]
Thanks @feed3r

**Pull request merged**